index.html

```diff
@@ -217,8 +217,6 @@ <h1 id="ovningar">Övningar</h1>
     <script src="https://ajax.googleapis.com/ajax/libs/jquery/2.1.1/jquery.min.js"></script>
     <script src="vendor/highlight.pack.js"></script>
     <script src="vendor/bootstrap.min.js"></script>
-    <script src="vendor/smooth-scroll.js"></script>
```

Comment:
Attaching scrolling when clicking on anchor links :) This way of smooth scrolling is a very common technique on the web, I don't really know why we're discussing it tbh.

How is it spaghetti? Honestly curious. It's a single, static page, with some JS dependencies? Shaving off tiny scripts won't do much good, and honestly: will it matter for the audience (the students reading)?